server/pages/index.js

```diff
@@ -370,11 +371,18 @@ module.exports = function() {
 			}
 		} );
 
-	app.get( '/browsehappy', function( request, response ) {
-		response.status( 200 ).render( 'browsehappy.jade', {
-			urls: generateStaticUrls( request )
+	app.get( '/browsehappy', setUpRoute, function( req, res, next ) {
+		const wpcomRe = /^https?:\/\/[A-z_-]+\.wordpress\.com$/;
+		const primaryBlogUrl = get( req, 'context.user.primary_blog_url', '' );
```

Comment:
Since the `context` object is passed wholesale to the .jade file (and its members made available there as globals of sort), I was actually thinking of putting that check (sans lodash) there.